### PR TITLE
decelerate!

### LINKED ISSUE #23184: Remove deprecated 'accelerate' connection plugin and associated keywords
##### ISSUE TYPE

 - Bug Report

##### COMPONENT NAME

accelerate

##### ANSIBLE VERSION

```
2.4
```

### PR DESCRIPTION
**Pull request by bcoca**
##### SUMMARY

bye accelerate
fixes #23184
##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT NAME

accelerate

##### ANSIBLE VERSION

```
2.5
```

### CODE REVIEW

**Comment by mattclay** [comment]
bot_status

**Comment by mattclay** [comment]
CI failure due to merge conflict.

**Comment by dagwieers** [comment]
Nice title :-)

**Review by dagwieers — APPROVED** [review]

File: lib/ansible/utils/encrypt.py [hunk 1]
```diff
@@ -1,28 +1,10 @@
...
-# (at your option) any later version.
-#
-# Ansible is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+# (c) 2017 Ansible Project
```
Comment (dagwieers) [inline lib/ansible/utils/encrypt.py:2]: FYI According to the big rewrite the new format for copyright statements is:
```python
# Copyright: (c) 2012-2015, Michael DeHaan <michael.dehaan@gmail.com>
# Copyright: (c) 2017, Ansible Project
```

**Pull request merged**